display_information/sketch-on-map/src/main/resources/sketch_on_map/main.fxml

```diff
@@ -25,7 +25,7 @@
 <?import javafx.scene.image.Image?>
```

Comment:
I seem to be getting an exception when I run this sample, can you check

Reply:
Found a quick fix, pushing the commit.